### PR TITLE
[INLONG-8964][Manager] Support Sortstandalone cluster management

### LINKED ISSUE #8964: [Feature][Manager] Support Sortstandalone cluster management
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #8964 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/sortstandalone/SortstandaloneClusterDTO.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    /**
+     * Get the dto instance from the JSON string.
+     */
+    public static SortstandaloneClusterDTO getFromJson(@NotNull String extParams) {
+        try {
+            return JsonUtils.parseObject(extParams, SortstandaloneClusterDTO.class);
+        } catch (Exception e) {
+            throw new BusinessException(ErrorCodeEnum.CLUSTER_INFO_INCORRECT,
+                    String.format("parse extParams of Pulsar Cluster failure: %s", e.getMessage()));
```
Comment (fuweng11) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/sortstandalone/SortstandaloneClusterDTO.java]: Sort stand alone

Comment (vernedeng) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/sortstandalone/SortstandaloneClusterDTO.java]: fixed

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/sortstandalone/SortstandaloneClusterDTO.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import javax.validation.constraints.NotNull;
+
+import java.util.Set;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@ApiModel("SortStandalone cluster info")
+public class SortstandaloneClusterDTO {
```
Comment (fuweng11) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/sortstandalone/SortstandaloneClusterDTO.java:42]: SortStandaloneClusterDTO

**Review by fuweng11 — APPROVED** [review]

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/sortstandalone/SortstandaloneClusterDTO.java (hunk 2 above)
Comment (vernedeng) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/sortstandalone/SortstandaloneClusterDTO.java]: fixed

**Review by lvanneo — APPROVED** [review]
LGTM

**Pull request merged**